spec/predicates/vuln.md

```diff
@@ -0,0 +1,170 @@
+> This field describes which scanner is used while performing a container image scan,
+> as well as version information and which Vulnerability DB is used.
+
+**scanner.uri** string (ResourceURI), optional
+
+> > URI indicating the identity of the source of the scanner.
+
+**scanner.version** string (ResourceURI), optional
+
+> > The version of the scanner.
+
+**scanner.db.uri** string (ResourceURI), optional
+
+> > > URI indicating the identity of the source of the Vulnerability DB.
+
+**scanner.db.version** string, optional
+
+> > > The version of the Vulnerability DB.
+
+**scanner.db.lastUpdate, required** string (timestamp)
+
+> > > The timestamp of when the vulnerability DB was updated last time.
+
+**scanner.result, required** list
+
+> > The result contains a list of vulnerabilities. Note that an empty list means the **scanner** found no vulnerabilities.
+> > This is the most important part of this field because it'll store the scan result as a whole. So, people might want
+> > to use this field to take decisions based on them by making use of Policy Engines tooling whether allow or deny these images.
+
+**scanner.result.[*].vulnerability** object, optional
+
+> > > The vulnerability object defines information about each one of the vulnerabilities found by the scanner.
+
+**scanner.result.[*].vulnerability.id, required** string
+
+> > > > This is the identifier of the vulnerability, e.g. [GHSA-fxph-q3j8-mv87](https://github.com/advisories/GHSA-fxph-q3j8-mv87) whose CVE id is [CVE-2017-5645](https://nvd.nist.gov/vuln/detail/CVE-2017-5645).
+
+**scanner.result.[*].vulnerability.severity, required** object
+
+> > > > The severity contains a list to describe the severity of a vulnerability using one or more quantitative scoring method.
+
+**scanner.result.[*].vulnerability.severity.method, required** string
+
+> > > > > The method describes the quantitative method used to calculate the associated severity score such as nvd, cvss and others.
+
+**scanner.result.[*].vulnerability.severity.score, required** string
+
+> > > > > This is a string representing the severity score based on the selected method.
+
+**scanner.result.[*].vulnerability.annotations** list, optional
+
+> > > > > This is a list of key/value pairs where scanners can add additional custom information.
+
+**metadata.scanStartedOn, required** string (timestamp)
+
+> > The timestamp of when the scan started.
+
+**metadata.scanFinishedOn, required** string (timestamp)
+
+> > The timestamp of when the scan completed.
+
+## Example
+
+```jsonc
+{
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [
+    {
+      "name": "foo.jar",
+      "digest": {"sha256": "fe4fe40ac7250263c5dbe1cf3138912f3f416140aa248637a60d65fe22c47da4"}
+    }
+  ],
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/vulns/v0.1",
+  "predicate": {
+    "invocation": {
+      "parameters": [],
+      "uri": "",
+      // https://github.com/developer-guy/alpine/actions/runs/1071875574
```

Comment:
I think these comments still need to get inlined in the example?

E.g. `"uri": "https://github.com/developer-guy/alpine/actions/runs/1071875574"`

Right?